README.md

```diff
@@ -4,7 +4,7 @@
      alt="Mockito" />
 </a>
 
-Most popular mocking framework for Java
+Most popular mocking framework for Java.
```

Comment:
Nit: let's leave the dot here, as this is our tagline

Reply:
Thank you @TimvdLippe, its resolved now.